share/completions/pnpm.fish

```diff
@@ -0,0 +1,20 @@
+function _pnpm_completion
+  set cmd (commandline -o)
+  set cursor (commandline -C)
+  set words (count $cmd)
+
+  set completions (eval env DEBUG=\"" \"" COMP_CWORD=\""$words\"" COMP_LINE=\""$cmd \"" COMP_POINT=\""$cursor\"" pnpm completion -- $cmd)
+
+  if [ "$completions" = "__tabtab_complete_files__" ]
+    set -l matches (commandline -ct)*
+    if [ -n "$matches" ]
+      __fish_complete_path (commandline -ct)
+    end
+  else
+    for completion in $completions
+      echo -e $completion
```

Comment:
That `-e` is suspect - this will re-evaluate backslash-escapes you got in the output of `pnpm completion`!

I doubt pnpm expects its output to be treated like that.